routes/users.js

```diff
@@ -3,7 +3,18 @@ const router = express.Router();
 const ObjectID = require("mongodb").ObjectID;
 const multer  = require('multer');
 const upload = multer({ dest: 'uploads/' });
-const { uuidv4 } = require('uuidv4');
+const { uuid } = require('uuidv4');
```

Comment:
thx for fixing lol

Reply:
have you tested it with registration?

Reply:
I tested it with a new registration, it seems to work.